docs/release-notes/8.2.0-1.md

```diff
@@ -0,0 +1,84 @@
+# Percona Server for MySQL 8.2.0-1 (2024-01-)
+
+Get started with [Quickstart Guide for Percona Server for MySQL].
+
+[Percona Server for MySQL 8.2.0-1] includes all the features and bug fixes available in the
+MySQL 8.2 Community Edition in addition to enterprise-grade features developed by Percona.
+
+## Release highlights
+
+This release merges the MySQL 8.2 code base. Withing this merge Percona introduces several sagnificant changes:
+
+* Removes remainders of Percona-specific encryption features (old 5.7 encrypted bilnog compatibility).
+
+* Removes the deprecated `Extended mysqlbinlog` tool.
```

Comment:
@oleksandr-kachan Please review/update this statement.

Reply:
@alina-derkach-oaza just remove this line completely

Reply:
fixed